PukingPredator/Assets/Scripts/GameManager.cs

```diff
@@ -91,6 +90,9 @@ public static void TransitionToScene(string sceneName)
             AudioManager.Instance.PlayBackground(AudioManager.MusicName.Title);
         }
 
-        SceneManager.LoadScene(sceneName);
+        if (isLevelScene)

```

Comment:
this is not how this should behave. we need a bandaid fix for now if youre going between the main menu and level select because those should really be the same scene, but every other case should have a transition. so if its that particular swap in either direction, it should be load scene, but otherwise it should be fadetoscene